### ISSUE TITLE
Put vertex "asymmetry coloration" back in

### ISSUE THREAD

**Issue by jeanbellynck** [jeanbellynck/Cayley-Graph-Generator#21]
way back, vertices were colored in red if their angles between two generators differed from the average angles between those two generators.

**Comment by p4jo** [comment]
I implemented this stress value as max_s(angle between in and outgoing edges for generator s)

**Comment by p4jo** [comment]
Close if this is enough

**Comment by jeanbellynck** [comment]
The aim of this issue is different. The idea is to make the places where the torus changes orientation more visible

**Comment by p4jo** [comment]
Yes, this requires implementing a global bookkeeping about the average angles, which can be done. Put angle properties into the vertex class and an averaging into the graph manager class
